src/main/java/com/iota/iri/MilestoneTracker.java

```diff
@@ -285,4 +290,14 @@ public void reportToSlack(final int milestoneIndex, final int depth, final int n
             e.printStackTrace();
         }
     }
+
+    private boolean isMilestoneBundleStructureValid(List<TransactionViewModel> bundleTxs, int securityLevel) {
```

Comment:
It is usually a good practice to have as an input parameter the least specific abstraction (`List -> `Collection`) the code can handle.
In contrast the return type should be specific, but still preferably an abstraction (interface).

Since this method is `private` it matters less.